plugins/zoom/help.md

```diff
@@ -101,8 +104,8 @@ This action is used to delete or disassociate user from account.
 |Name|Type|Default|Required|Description|Enum|Example|
 |----|----|-------|--------|-----------|----|-------|
 |action|string|None|True|Specify how to delete the user. To delete pending user, use disassocaite|['delete', 'disassociate']|delete|
-|id|string|None|True|The user identifier or email address|None|user@example.com|
-|transfer_email|string|None|False|Email to transfer meetings, webinars, or recordings|None|user@example.com|
+|id|string|None|True|The user identifier or email address|None|https://example.com|
```

Comment:
Looks like email addresses throughout need to be updated as they have gone to urls.